source/common/http/alternate_protocols_cache_impl.h

```diff
@@ -83,9 +91,7 @@ class AlternateProtocolsCacheImpl : public AlternateProtocolsCache,
   getOrCreateHttp3StatusTracker(const Origin& origin) override;
 
 private:
-  void setAlternativesImpl(const Origin& origin, std::vector<AlternateProtocol>& protocols);
   void setSrttImpl(const Origin& origin, std::chrono::microseconds srtt);
```

Comment:
I'm slightly surprised that this method is still here given the discussion which seemed to suggest we were settling on a single setImpl() method?

Reply:
fixed